server/service/service_osquery_options.go

```diff
@@ -1,41 +1,20 @@
 package service
 
 import (
-	"github.com/ghodss/yaml"
```

Comment:
Is this no longer being depended on at all? If so, we can probably re-run `dep` to remove it from the lockfile.

Reply:
It's still used in the test (to verify the correct JSON tags) and I think it should remain for use in the client.